tests/sentry/incidents/test_metric_issue_post_process.py

```diff
@@ -85,7 +85,7 @@ def test_occurrence_post_process(self, mock_process_workflows):
         result: dict[DetectorGroupKey, DetectorEvaluationResult] = self.handler.evaluate(
             data_packet
         )
-        evaluation_result: DetectorEvaluationResult = result[self.detector_group_key]
```

Comment:
I'm not entirely sure which part of my change affected the return value like this, will revisit tomorrow